apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -406,18 +407,18 @@ static Extension createExtensionDate(
       CcwCodebookInterface ccwVariable, Optional<BigDecimal> dateYear) {
 
     Extension extension = null;
+    if (!dateYear.isPresent()) {
+      throw new NoSuchElementException();
+    }
```

Comment:
I would argue that the fact that we are doing this implies that the dateYear is really not optional and we should change this method to just take a BigDecimal (possiblly with a lombok NonNull annotation if we decide to go that route). However that is a refactoring outside of what you are trying to accomplish. Thank you for doing this and at least making it explicit.